### ISSUE TITLE
Question: Clarification on max_evals

### ISSUE THREAD

**Issue by daradermody** [regel/loudml#63]
I understand that `max_evals` is the "number of iterations to produce a model with optimal accuracy", but I'm not sure how this value should be chosen. I would guess that the higher this value, the longer the model takes to train, but does accuracy remain consistent with the same `max_eval` number between use cases (differing bucket size, signal period, signal noise, etc.)? If it does, how would you suggest I quantitatively determine what number is good for me (seeing accuracy vs training time)? I am only developing a Kibana plugin on top of it, so I can't predict the variety of use cases and data characteristics that will be used.

Or maybe you could tell me what a good value is? :smiley_cat:

**Comment by regel** [comment]
Hi @daradermody, we're trying to make the default values work in most situations. If unsure, one option is to omit this value in the standard settings, and have it somewhere in expert settings. `max_evals` controls the hyper-parameters search ie finding the ANN topology that best fits the training data-set.

**Comment by daradermody** [comment]
I'm not sure what you mean by standard vs expert settings. Could you clarify?

**Comment by regel** [comment]
`max_evals` is difficult to explain in the GUI for first time users, it's one of the FAQ. The ability to choose 'auto' can be an option if the user is unsure about the value to input in this setting.
